api/src/main/resources/workbench-api.yaml

```diff
@@ -249,6 +249,16 @@ paths:
       responses:
         204:
           description: Request Received.
+  "/v1/confirmProfile":
+    post:
+      tags:
+        - profile
+      description: Used by users to confirm that their profile is up to date, as part of the
+        annual renewal compliance process.
+      operationId: confirmProfile
+      responses:
+        204:
+          description: Request Received.
```

Comment:
Perhaps, "Confirmed." "Request received" makes it sound like the update will happen sometime in the future.

That said, I'd prefer you merge this change as-is rather than fix this doc if it means you'd have to run all of the tests again.

Reply:
copypasta.  Thanks for the catch.

Someone snuck in a Liquibase change before me so I'll have to rebase anyway.